packages/frontend/src/components/SearchBar.tsx

```diff
@@ -1,4 +1,4 @@
-import { useRef, useState } from "react";
+import { useEffect, useRef, useState } from "react";
```

Comment:
remove the unused import. This will be an eslint error once CI is back online. I would recommend installing the Microsoft eslint plugin if you are using vscode